cmd/rizzod/main.go

```diff
@@ -293,7 +293,7 @@ func lockComment() (string, error) {
 }
 
 func parseStderr(oid string, stderr string) (*rizzopb.PuppetReport, error) {
-	var regexEvalErr = regexp.MustCompile(`^Error: Evaluation Error:`)
+	var regexEvalErr = regexp.MustCompile(`^Error: (?:Evaluation|Syntax) [Ee]rror`)
```

Comment:
I think you will want a separate regex, since otherwise you can't differentiate the type of error returned in line 302?

Reply:
The problem with that is that it would require a more significant code change, as there are a bunch of places that heckler explicitly looks for `EvalError`. I was thinking this would be a good stopgap for now, as the two errors are often indicative of the same problem anyway (typo in the code), with a plan to follow up with a more substantial fix in the future.

Reply:
true, seems like a fine 🩹